drawframe.py

```diff
@@ -74,12 +74,10 @@ def __init__(self, x, y):
 
 
 def place_scenery(canvas, y):
-    x = 0
-    for block in range(int(canvas.width)):
+    for x in range(0, canvas.width, BLOCK_SIDE):
```

Comment:
FYI this was looping too many times. At the end, x would be SCREEN_WIDTH * BLOCK_SIDE.